Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyCreateProduct.Codeunit.al

```diff
@@ -157,6 +127,7 @@ codeunit 30174 "Shpfy Create Product"
                 ItemUnitofMeasure.SetRange("Item No.", Item."No.");
                 if ItemUnitofMeasure.FindSet(false) then
                     repeat
+                        TempShopifyProduct."Has Variants" := true;
```

Comment:
This was a Bug. Product was created on Shopify with variants (due to UOM), yet in BC it was marked as if it doesn't have any.